### ISSUE TITLE
ConsumerGroupStream: commit with force=true will break future commits with force=false

### ISSUE THREAD

**Issue by navbruce** [SOHU-Co/kafka-node#1065]
## Bug Report

Hopefully a reasonably self-evident logic bug:

In consumerGroupStream.js, if `commit(msg, force=true, cb)` is called (with a valid message & offset) it will in turn call `commitQueued(commits, force=true, cb)`.  `commitQueued()` then sets `this.committing=true`, but skips the `setTimeout()` call that is required to set `this.committing=false`.

Any future calls to `commit(msg, force=false, cb)` will bail out on the `if(this.committing && !force)` test, and therefore `commitQueued()` will not be called.

Basically, `this.committing=true` should only happen if the `setTimeout()` call will happen, or else `this.committing` will never be reset to false.

The end result of this bug is that offsets are never being auto-committed on a topic I'm consuming with ConsumerGroupStream.

PR: https://github.com/SOHU-Co/kafka-node/pull/1066

### Environment

* Node version: 8.11.4
* Kafka-node version: 2.6.1, 3.0.0
* Kafka version: 1.1.0
